src/chainparams.cpp

```diff
@@ -263,10 +264,10 @@ class CTestNetParams : public CChainParams {
         consensus.vDeployments[Consensus::DEPLOYMENT_SEGWIT].nTimeout = 0; // Disabled
 
         // The best chain should have at least this much work.
-        consensus.nMinimumChainWork = uint256S("0x00000000000000000000000000000000000000000000000000001030d1382ade");
+        consensus.nMinimumChainWork = uint256S("000000000000000000000000000000000000000000000000003e3c33bc605e5d"); // 3,976,284
```

Comment:
I missed a `0x` prefix here. Not blocking but sloppy, so I will update this line.

Reply:
Done: [change after force push]( https://github.com/dogecoin/dogecoin/compare/0d3c023354e4a4e532fafc3da7c7ae472e221802..43c0008fb79dffa5971264c04aecc06c4576fa33)